go/cmd/vtctldclient/command/onlineddl.go

```diff
@@ -83,6 +86,22 @@ var (
 		Args:                  cobra.ExactArgs(2),
 		RunE:                  commandOnlineDDLRetry,
 	}
+	OnlineDDLThrottle = &cobra.Command{
+		Use:                   "throttle <keyspace> <uuid|all>",
+		Short:                 "throttles one or all migrations",
```

Comment:
nit: these are full sentences with capitalization/punctiuations for some of the others

Reply:
(see `Show` and `Cleanup`)

Reply:
fixed